dev/com.ibm.ws.grpc_fat/publish/files/grpc.client.security.plaintext.xml

```diff
@@ -0,0 +1,47 @@
+<!--
+    Copyright (c) 2020 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+   
+    Contributors:
+        IBM Corporation - initial API and implementation
+ -->
+<server description="Basic gRPC with security constraints">
+    <featureManager>
+        <feature>grpc-1.0</feature>
+        <feature>grpcClient-1.0</feature>
+        <feature>appSecurity-2.0</feature>
+    </featureManager>
+    
+    <grpcClient host="*" sslRef="CustomSSLSettings" usePlaintext="true" />
```

Comment:
This is correct - the `usePlaintext="true"` cancels out the `sslRef="CustomSSLSettings"` - but this will be confusing to anyone looking at this test in the future.  Can you additionally remove the `sslRef`, or add a comment about these canceling out?